src/transformers/models/squeezebert/configuration_squeezebert.py

```diff
@@ -154,3 +157,19 @@ def __init__(
         self.post_attention_groups = post_attention_groups
         self.intermediate_groups = intermediate_groups
         self.output_groups = output_groups
+
+
+class SqueezeBertOnnxConfig(OnnxConfig):
```

Comment:
Could you please add this to ensure we propagate changes in the source model (should they arise):

```suggestion
# # Copied from transformers.models.bert.configuration_bert.BertOnxxConfig with Bert->SqueezeBert
class SqueezeBertOnnxConfig(OnnxConfig):
```